lib/custom-resource-manager.js

```diff
@@ -104,6 +104,20 @@ class CustomResourceManager {
 
     throw new Error(`Unable to update resource ${resourceName}`)
   }
+
+  async manageCrd ({ customResourceManifest }) {
+    this._kubeClient.addCustomResourceDefinition(customResourceManifest)
```

Comment:
Does this always need to be added, perhaps only do this if its not disabled? I'm thinking it would be discovered by `await kubeClient.loadSpec()` in daemon above if CRDs are already in the cluster.